pages/pipelines/job_minutes.md.erb

```diff
@@ -22,9 +22,9 @@ The [Usage page](https://buildkite.com/organizations/~/usage) is available on ev
 
 <%= image "usage.png", width: 554, height: 794, alt: "Job minute usage page" %>
 
-## Limits for the free plan
+## Limits for the Developer Plan
 
-The free plan has a limit of 10,000 job minutes per month. When this limit is reached, builds are not cancelled, but jobs will not be run. The limit is reset when your billing period rolls over, or your organization is upgraded to a different plan.
+The Developer Plan has a limit of 5,000 job minutes per month. When this limit is reached, builds are not cancelled, but jobs will not be run. The limit is reset when your billing period rolls over, or your organization is upgraded to a different plan.
 
 
 ### Build timeouts
```

Comment:
Ah, sorry actually I found another mention of "free" that hasn't been updated. See the first sentence under Build timeouts (line 32 https://github.com/buildkite/docs/pull/1773/files#diff-e42f6cb39bab6ce83ca8f3d58fd4ffb498fb61c0035c3b33ac9ca3601a45e992R32 )

Should be:
> Developer plans have a…

Reply:
nice spotting!